models/topicsModel.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection");
+
+exports.getAllTopics = () =>{
+    return db.query("SELECT * FROM topics")
```

Comment:
Ideally you should be returning your rows here so that you can turn it into an object in the controller.